cuebot/src/main/resources/opencue.properties

```diff
@@ -44,6 +44,48 @@ dispatcher.job_frame_dispatch_max=8
 # Maximum number of frames to dispatch from a host at one time.
 dispatcher.host_frame_dispatch_max=12
 
+# Number of threads to keep in the pool for launching job.
+dispatcher.launch_queue.core_pool_size=1
+# Maximum number of threads to allow in the pool for launching job.
+dispatcher.launch_queue.max_pool_size=1
+# Queue capacity for launching job.
+dispatcher.launch_queue.queue_capacity=100
+
+# Number of threads to keep in the pool for various operation.
+dispatcher.dispatch_pool.core_pool_size=4
+# Maximum number of threads to allow in the pool for various operation.
+dispatcher.dispatch_pool.max_pool_size=4
+# Queue capacity for various operation.
+dispatcher.dispatch_pool.queue_capacity=500
```

Comment:
The original values
https://github.com/AcademySoftwareFoundation/OpenCue/blob/87bcf7ab6eb246afb70e7259c7d9f7946f556bf5/cuebot/src/main/resources/conf/spring/applicationContext-service.xml#L64-L66